motr/client_init.c

```diff
@@ -1678,16 +1683,35 @@ int m0_client_init(struct m0_client **m0c_p,
 	}
 
 	if (conf->mc_is_addb_init) {
-		char buf[64];
+		char        buf[128];
```

Comment:
in the previous commit the buf size was increased to 255. Any reason for reducing it again?